.gitignore

```diff
@@ -9,3 +9,4 @@ ISSUES.md
 /vendor/gems
 /coverage
 .idea/
+TAGS
```

Comment:
it is the default [ctags][0] file for emacs. `tags` (all lowercase) is the default ctags file for vim

[0]:https://blog.sensible.io/2014/05/09/supercharge-your-vim-into-ide-with-ctags.html